addons/l10n_br/i18n/l10n_br.pot

```diff
@@ -0,0 +1,7280 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* l10n_br
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 16.1alpha1+e\n"
```

Comment:
It looks like the file has some issues (according to poedit)
Could you please have a look? It's probably nothing important
Thank you :)

Reply:
I think it should be ok now :slightly_smiling_face: